### PR TITLE
Add Nix flake

### PR DESCRIPTION
**Pull request by joncol**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  description = "Flake for openapi2slate";
+
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+  inputs.flake-utils = {
+    url = "github:numtide/flake-utils";
+    inputs.nixpkgs.follows = "nixpkgs";
```
Comment (zlondrej) [inline flake.nix]: What's the point of this line?

`inputs.nixpkgs.follows = "nixpkgs";`

With it, it gives warning and it seems to work without it.
> warning: input 'flake-utils' has an override for a non-existent input 'nixpkgs'

So it could be just `inputs.flake-utils.url = "github:numtide/flake-utils";`

**Review by zlondrej — CHANGES_REQUESTED** [review]

File: flake.nix (hunk 1 above)
Comment (joncol) [reply flake.nix]: Good point, something I had in my flake template for unknown reasons. Removed it.

File: test/check/scrive_api.md [hunk 2]
```diff
@@ -5416,7 +5416,7 @@ Similarly, a single field can have multiple placements on the document.</p>
 <p><strong>Note:</strong> Some field types have <em>no effect</em> without at least one placement.</p>
 
 
-<p>The value of this array must match <em>at least one</em> of the following schemas:</p>
+<p>Each element of this array must match <em>at least one</em> of the following schemas:</p>
```
Comment (zlondrej) [inline test/check/scrive_api.md]: This is unrelated to this PR. It should go away if you merge https://github.com/scrive/openapi2slate/pull/16 first though.

**Review by zlondrej — CHANGES_REQUESTED** [review]

File: test/check/scrive_api.md (hunk 2 above)
Comment (joncol) [reply test/check/scrive_api.md]: Exactly. Created the PR branch as a "spinoff" branch to avoid having to merge/rebase later.

File: CHANGELOG.md [hunk 3]
```diff
@@ -14,6 +14,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   > Each element of this array must match *at least one* of the following
   > schemas
 
+- Create Nix flake
```
Comment (zlondrej) [inline CHANGELOG.md]: Well, it's not really library version relevant. That's just development tool.

Comment (joncol) [reply CHANGELOG.md]: Removed.

**Review by zlondrej — APPROVED** [review]
